websites/V/VidCloud9/presence.ts

```diff
@@ -23,114 +31,120 @@ presence.on("iFrameData", (data) => {
   }
 });
 
-function getTimestamps(videoTime, videoDuration): any {
-  var startTime = Date.now();
-  var endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
+function getTimestamps(videoTime: number, videoDuration: number): Array<number> {
+  const startTime = Date.now(), endTime = Math.floor(Math.floor(startTime / 1000) - videoTime + videoDuration);
   return [Math.floor(startTime / 1000), endTime];
 }
 
 presence.on("UpdateData", async () => {
-  const info = await presence.getSetting("sSI");
-  const elapsed = await presence.getSetting("sTE");
-  const videoTime = await presence.getSetting("sVT");
+  const info = await presence.getSetting("sSI"), elapsed = await presence.getSetting("sTE"), videoTime = await presence.getSetting("sVT");
 
   if (elapsed) {
-    var browsingStamp = Math.floor(Date.now() / 1000);
-    console.log("Elapsed is on");
+    browsingStamp = Math.floor(Date.now() / 1000);
+    presence.info("Elapsed is on");
```

Comment:
presence.info is only available when you will enable them in metadata by ``readLogs`` to true

Reply:
If that's the case then `readLogs` is true by default.

Reply:
a no wait lol ,you are right 🤦 . readlogs are only for getting logs bruh

Reply:
Fixed guidelines btw about it lol

Reply:
cool